benchmark/hyperqueue/benchmark.py

```diff
@@ -0,0 +1,33 @@
+import subprocess as sp
+import time
+
+from datetime import datetime
+
+
+hq_bin = "/home/user3574/PycharmProjects/hyperqueue/target/release/hq"
+
+
+def hq_cmd(cmd, samples=1):
+    start_time = datetime.now()
+    for sample in range(samples):
+        sp.call([hq_bin + ' ' + cmd], shell=True)
+    sp.call([hq_bin + ' ' + "wait 1-10"], shell=True)
+    end_time = datetime.now()
+    return format(end_time - start_time)
+
+
+def benchmark(cmd, samples, data):
+    # Start
+    sp.call([hq_bin + ' ' + "server start &"], shell=True)
+    sp.call([hq_bin + ' ' + "worker start &"], shell=True)
+    time.sleep(0.1)
+
+    time_taken = hq_cmd(cmd, samples)
+    print('Execution of: ' + cmd + '\nTime taken: ' + str(time_taken))
```

Comment:
You can use f-strings to make formatting more readable:
`print(f"Execution of {cmd}\nTiem taken: {time_taken}")`